firefox_puppeteer/tests/test_utils.py

```diff
@@ -0,0 +1,74 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness import FirefoxTestCase
+
+
+class TestUtils(FirefoxTestCase):
```

Comment:
As I have mentioned on the other bug you want to name this class similar to the method you are testing. So in this case `TestSanitize`.